Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -862,6 +866,8 @@ - (void)testSpecTests {
       }
     }];
   }
+  NSLog(@"%@ completed; pass=%@ skip=%@", NSStringFromClass([self class]), @(testPassCount),
```

Comment:
Now that these values are plain ints, you can make the format `%d` and pass the value without boxing.

Reply:
Done.